pkg/catalog/helm/helm.go

```diff
@@ -44,7 +67,7 @@ func DownloadIndex(indexURL string) (*RepoIndex, error) {
 	defer resp.Body.Close()
 
 	if resp.StatusCode != http.StatusOK {
```

Comment:
you should only return forgot error if status code is unauthorized.

Reply:
updated, thanks.